.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ executors:
   # also be updated.
```

Comment:
would be cool to update the prometheus orb too: `prometheus/prometheus@0.16.0`

Reply:
Done.l